src/Microsoft.IdentityModel.Tokens.Saml/Saml/SamlAuthorizationDecisionStatement.cs

```diff
@@ -77,39 +91,54 @@ internal SamlAuthorizationDecisionStatement()
             CheckObjectValidity();
         }
 
-        public static string ClaimType
+        // TODO can this be null ???
+        /// <summary>
+        /// Gets or sets the access decision contained in the AuthorizationDecisionStatement.
+        /// </summary>
+        public SamlAccessDecision AccessDecision
         {
-            get
-            {
-                return ClaimTypes.AuthorizationDecision;
-            }
+            get; set;
         }
 
+        /// <summary>
+        /// Gets or sets the action values contained in the AuthorizationDecisionStatement.
+        /// </summary>
         public ICollection<SamlAction> Actions
         {
-            get { return _actions; }
+            get;
         }
 
-        // TODO can this be null ???
-        public SamlAccessDecision AccessDecision
+        /// <summary>
+        /// Gets or sets the ClaimType.
```

Comment:
We need set this value when reading xml with SamlSerializer. Change to internal set.